spring-cloud-aws-dependencies/pom.xml

```diff
@@ -33,7 +33,7 @@
 		<aws-java-sdk.version>1.11.336</aws-java-sdk.version>
 		<elasticache.version>1.1.1</elasticache.version>
 		<jmemcached.version>1.0.0</jmemcached.version>
-		<spring-cloud-context.version>1.3.2.RELEASE</spring-cloud-context.version>
+		<spring-cloud-context.version>2.0.0.RELEASE</spring-cloud-context.version>
```

Comment:
That's indeed nasty bug 😢 .

I wonder why `parent` `<artifactId>spring-cloud-dependencies-parent</artifactId>` doesn't do the trick to resolve the version for the `spring-cloud-context` automatically...

Reply:
As far as I can see there are no dependency versions defined in spring-cloud-dependencies-parent

Reply:
OK. I see.
Now the question: why this `spring-cloud-context` here in the BOM at all?
It is not a part of this project and we don't manage here its version. It has to come to the target project via the proper `spring-cloud-dependencies` BOM.

I mean that I suggest to remove this dependency from here.

Reply:
When I try to remove it:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.springframework.cloud:spring-cloud-context:jar is missing. @ org.springframework.cloud:spring-cloud-aws-parameter-store-config:[unknown-version], /Users/ziembam/Workspace/Code/spring-cloud-aws/spring-cloud-aws-parameter-store-config/pom.xml, line 42, column 15
```

so looks like this version is used in spring-cloud-aws-parameter-store-config

Reply:
I see. So, that one uses correct:
```
<parent>
		<groupId>org.springframework.cloud</groupId>
		<artifactId>spring-cloud-aws</artifactId>
		<version>2.0.2.BUILD-SNAPSHOT</version>
</parent>
```
but what is missed there is something like this:
```
<dependency>
				<groupId>org.springframework.cloud</groupId>
				<artifactId>spring-cloud-commons-dependencies</artifactId>
				<version>${spring-cloud-commons.version}</version>
				<type>pom</type>
				<scope>import</scope>
</dependency>
```
Which really will for us that `spring-cloud-context`, but again: we must not provide such a dependency in our BOM.

Reply:
I've move the spring-cloud-context dependency from spring-cloud-aws-dependencies BOM to spring-cloud-aws parent pom